src/main/scala/chisel3/stage/phases/Elaborate.scala

```diff
@@ -13,21 +13,26 @@ import chisel3.stage.{
   ThrowOnFirstErrorAnnotation
 }
 import firrtl.AnnotationSeq
-import firrtl.options.Phase
+import firrtl.options.{Dependency, Phase}
 import firrtl.options.Viewer.view
+import logger.LoggerOptions
 
 /** Elaborate all [[chisel3.stage.ChiselGeneratorAnnotation]]s into [[chisel3.stage.ChiselCircuitAnnotation]]s.
   */
 class Elaborate extends Phase {
 
-  override def prerequisites = Seq.empty
+  override def prerequisites: Seq[Dependency[Phase]] = Seq(
+    Dependency[chisel3.stage.phases.Checks],
+    Dependency(_root_.logger.phases.Checks)
```

Comment:
This brings in a transitive dependency on `_root_logger.phases.AddDefaults` which will cause a default `LogLevelAnnotation` to get pulled in.

I think that `Check` should probably _not_ have `AddDefaults` as a prerequisite and instead it should be an optional one. I'll see about changing this.

Reply:
This is probably not the issue. The real issue is that the logger annotations shouldn't be emitted. I'm testing https://github.com/chipsalliance/chisel/pull/3901 which marks them unserializable and prevents them from showing up in the output CHIRRTL.